lib/Installation/Partitioner/LibstorageNG/v4_3/SelectDisksToUsePage.pm

```diff
@@ -0,0 +1,56 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: The class introduces all accessing methods for Select Hard Disk(s)
+# Page in Guided Setup in case multiple disks are available in the system.
+
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package Installation::Partitioner::LibstorageNG::v4_3::SelectDisksToUsePage;
+use strict;
+use warnings FATAL => 'all';
+
+sub new {
+    my ($class, $args) = @_;
+    my $self = bless {
+        app => $args->{app}
+    }, $class;
+    return $self->init($args);
+}
+
+sub init {
+    my ($self) = shift;
+    $self->{btn_next}                = $self->{app}->button({id => 'next'});
+    $self->{lbl_select_disks_to_use} = $self->{app}->label({label => 'Select one or more (max 3) hard disks'});
+    return $self;
+}
+
+sub _get_disk_checkbox {
```

Comment:
I would leave it, as the way to identify the checkbox might change in future (e.g. new format) and then we can override just this method instead of whole `select_disk`. It also makes it more clear imho, that this control is dynamic.